WoopraSDK/WIdentify.swift

```diff
@@ -0,0 +1,76 @@
+import Foundation
+
+public class WIdentify {
+    
+    private let wIdnetifyEndPoint = "https://www.woopra.com/track/identify"
+    private let tracker: WTracker
+    
+    init(
+        tracker: WTracker
+    ) {
+        self.tracker = tracker
+    }
+    
+    func run() {
+        let url = URL(string: wIdnetifyEndPoint)!
+        var urlRequest = URLRequest(url: url)
+        urlRequest.httpMethod = "POST"
+        
+        var requestBody: [String: Any] = [
+            "host": tracker.domain ?? "",
+            "cookie": tracker.visitor.cookie,
+            "app": "ios",
+            "response": "xml",
+            "os": "ios",
```

Comment:
imo: I notice that in **WTracker.swift**, it sets the `os` field with the code `instance.add(property: "os", value: "\(UIDevice.current.systemName) \(UIDevice.current.systemVersion)")`. Should we use the same way to fill in the `os` field?

Reply:
@wubai-appier code updated!